content/pt/docs/concepts/instrumentation/libraries.md

```diff
@@ -0,0 +1,465 @@
+não operacionais. A menos que sua aplicação importe o SDK do OpenTelemetry, sua
+instrumentação não faz nada e não impacta o desempenho da aplicação.
+
+**Bibliotecas devem usar apenas a API do OpenTelemetry.**
+
+Você pode estar com receio de adicionar novas dependências, então aqui estão
+algumas considerações para ajudar a minimizar problemas com dependências:
+
+- A API de rastros do OpenTelemetry alcançou estabilidade no início de 2021,
+  seguindo a
+  [Convenção semântica 2.0](/docs/specs/otel/versioning-and-stability/), e
+  levamos a estabilidade da API a sério.
+- Ao definir dependências, use a versão estável da API do OpenTelemetry (1.0.\*)
+  e evite atualizá-la, a menos que precise de novos recursos.
+- Enquanto sua instrumentação se estabiliza, considere lançá-la como um pacote
+  separado, para que isso não cause problemas para usuários que não a utilizam.
+  Você pode mantê-la em seu repositório ou
+  [adicioná-la ao OpenTelemetry](https://github.com/open-telemetry/oteps/blob/main/text/0155-external-modules.md#contrib-components),
+  para que seja distribuída junto com outras bibliotecas de instrumentação.
+- As Convenções Semânticas são [estáveis, mas sujeitas à evolução][]: embora
+  isso não cause problemas funcionais, pode ser necessário atualizar sua
+  instrumentação de tempos em tempos. Ter a instrumentação em um pacote
+  experimental ou no repositório _contrib_ do OpenTelemetry pode ajudar a manter
+  as convenções atualizadas sem causar mudanças disruptivas para seus usuários.
+
+  [estáveis, mas sujeitas à evolução]:
+    /docs/specs/otel/versioning-and-stability/#semantic-conventions-stability
+
+### Obtendo um rastro
+
+Toda a configuração da aplicação é ocultada da sua biblioteca por meio da API de
+Rastros. As bibliotecas podem permitir que as aplicações passem instâncias de
+`TracerProvider` para facilitar a injeção de dependências e o teste, ou podem
+obtê-las a partir do
+[TracerProvider global](/docs/specs/otel/trace/api/#get-a-tracer). As
+implementações do OpenTelemetry em diferentes linguagens podem ter preferências
+distintas para passar instâncias ou acessar o global, dependendo do que é mais
+comum na linguagem.
+
+Ao obter o rastro, forneça o nome e a versão da sua biblioteca (ou do pacote de
+rastreamento) - essas informações aparecerão na telemetria e ajudarão os
+usuários a processar e filtrar a telemetria, além de entender sua origem e
+depurar/relatar quaisquer problemas de instrumentação.
+
+## O que instrumentar
+
+### APIs Públicas
+
+APIs públicas são bons candidatos para rastreamento: trechos criados para
+chamadas de APIs públicas permitem que os usuários mapeiem a telemetria para o
+código da aplicação, entendam a duração e o resultado das chamadas da
+biblioteca. Quais chamadas devem ser rastreadas:
+
+- métodos públicos que fazem chamadas de rede internamente ou operações locais
+  que levam tempo significativo e podem falhar (e.g. IO)
```

Comment:
ou mudar para itálico, por ser termo em inglês, ou mudar para `Entrada/Saída`

Reply:
Prefiro mudar para Entranda/Saída.